docs/deployment-arbitrary-cluster.md

```diff
@@ -0,0 +1,124 @@
+# Deployment on Arbitrary Cluster
+
+In this guide we will cover installation of Open vSwitch, NetworkManager and
+kubernetes-nmstate on your arbitrary cluster.
+
+## Requirements
+
+This guide requires you to have your own Kubernetes/OpenShift cluster. If you
+don't have one and just want to try kubernetes-nmstate out, please refer to
+[deployment on local cluster](deployment-local-cluster.md) guide.
+
+In order to get kubernetes-nmstate running, NetworkManager, and optionally
+Open vSwitch, must be installed on node. This requirement will be dropped once
+we make these components a part of kubernetes-nmstate container images.
+
+### NetworkManager
+
+kubernetes-nmstate container communicates with a NetworkManager instance running
+on the node using D-Bus. Make sure that NetworkManager is installed and running
+on each node.
+
+On CentOS:
+
+```shell
+yum install openvswitch
+systemctl start openvswitch
+```
+
+On Fedora:
+
+```shell
+dnf install openvswitch
```

Comment:
yum can be used on Fedora nowadays as well so you can simplify this to use the same commands for CentOS and Fedora. Btw. it seems to me that openvswitch is missing a dependency for libibverbs and therefore starting openvswitch might just fail.

Reply:
Above commands are for openvswitch but it is the NetworkManager section...

Reply:
Done